web/src/components/LastUpdateInfo.tsx

```diff
@@ -26,7 +26,7 @@ export const LastUpdateInfo = ({
   const { i18n, t } = useTranslation('common')
   return (
     <TimeStamp className={className}>
-      {withText && t('lastUpdate')} {lastUpdate.setLocale(i18n.language).toFormat(format)}
+      {withText && t('lastUpdate')} {withText && lastUpdate.setLocale(i18n.language).toFormat(format)}
```

Comment:
This changes the behavior of the prop. The prop is just there to decide whether to just display a date or a date and the text `last update:`.  A better option would be to just not render the whole component if you don't want it to show at all.